packages/components/src/components/Drawer/DrawerHeader.tsx

```diff
@@ -7,17 +7,21 @@ export interface DrawerHeaderProps
   extends Omit<React.HTMLAttributes<HTMLDivElement>, "color"> {
   /** The contents of the drawer header */
   children: NonNullable<React.ReactNode>;
+  padding?: "space0" | "space60";
 }
 
 /** The header content area of the drawer. */
 const DrawerHeader = React.forwardRef<HTMLDivElement, DrawerHeaderProps>(
-  ({ children, ...props }, ref) => {
+  ({ children, padding = "space60", ...props }, ref) => {
     return (
       <Box.div
         display="flex"
         gap="space30"
         justifyContent="space-between"
-        padding="space60"
+        paddingBottom={padding}
+        paddingLeft={padding}
+        paddingRight={padding}
+        paddingTop={padding}
```

Comment:
Since we are not setting individual padding we can keep this

```suggestion
        padding={padding}
```